src/profile-logic/call-tree.js

```diff
@@ -287,7 +287,12 @@ function _getInvertedStackSelfTimes(
   sampleCallNodes: Array<IndexIntoCallNodeTable | null>,
   interval: Milliseconds
 ): {
+  // In an inverted profile, the root node keeps all the self time
+  // So `callNodeSelfTime` will be 0 for all non-root node.
   callNodeSelfTime: Float32Array,
+  // This property stores the "amount of self time" spent in a specified call
+  // node (that's really a parent of the root). Later these values will make it
```

Comment:
wait, a parent of the root? I don't understand

Reply:
This is an inverted tree; so the "root" of this branch is actually a child in the uninverted tree. But I agree I had to read this several times to understand it, I should try to rephrase it.

Reply:
I think this comment is not very right actually, I'll rewrite it. Once I understand again ;)